report/firefox_code_coverage/codecoverage.py

```diff
@@ -113,17 +113,19 @@ def download_artifact(task_id, artifact, artifacts_path):
 
 
 def get_chunk(task_name):
```

Comment:
We really need to share some code between bot & report

Reply:
Yep, we have an issue on file for that (#237).